libsplinter/src/biome/datastore/diesel/models.rs

```diff
@@ -91,3 +91,21 @@ impl From<SplinterUser> for UserModel {
         UserModel { id: user.id() }
     }
 }
+
```

Comment:
Small typo in commit message: s/ grated / created ? Also, need to add a period at the end of the message.

Reply:
updated